lib/matplotlib/legend.py

```diff
@@ -428,7 +430,7 @@ def __init__(self, parent, handles, labels,
             The vertical offset (relative to the font size) for the markers
             created for a scatter plot legend entry. 0.0 is at the base the
             legend text, and 1.0 is at the top. To draw all markers at the
-            same height, set to ``[0.5]``. Default is ``[0.375, 0.5, 0.3125]``.
+            same height, set to ``[0.5]``. Default ``[0.375, 0.5, 0.3125]``.
```

Comment:
You changed this docstring here.  If you want to make the change, you also need to change it in `figure.py` and `axes/_axes.py`

Reply:
ah. okay. gotcha, ill keep it consistent to how the other files are.

Thanks again.

Reply:
Yes, see #10799 for the proposed fix to this lamentable state of affairs.  But in this case, I'd just remove the change.